docs/source/app-dev/app-arch.rst

```diff
@@ -148,6 +148,24 @@ To use command deduplication, you should:
 
 For more details on command deduplication, see the :ref:`Ledger API Services <command-submission-service-deduplication>` documentation.
 
+
+.. _handling-participant-node-failover:
+
+Handling participant node failover
+**********************************
+
+Some DAML Ledgers support exposing multiple eventually consistent Ledger API endpoints hosted by separate participant nodes.
```

Comment:
suggestion: change the 'hosted by separate participant nodes' into its own sentence and make it an example.

`Ledger API endpoints; e.g., by hosting separate participant nodes that expose the same view onto the ledger.`